common/locales/en/merch.json

```diff
@@ -0,0 +1,10 @@
+{
+  "merch" : "Merchandise",
+
+  "merch-0-summary" : "Teespring is a platform that makes it easy for anyone to create and sell high-quality products people love, with no cost or risk.",
```

Comment:
Hey @crookedneighbor 
Just updated this part, just checking something first i discussed with @Alys. 
I can make the links to the logo and the merchandise image dynamic like you suggested, but that mean when a image is added the name of the image has to be exactly typed that way. Is this something desirable? The way it is now, they just type over the image name and can verify the name is correct themselves. 
I can find myself in both options, when clear guidelines are presented i think the dynamic should be no problem. 

Just let me know and i'll put in what you guys think is best! 

Cheers (and a little to late but happy Thanksgiving! ;))